src/providers/fault-mgmt-providers/realtime-fault-provider.js

```diff
@@ -34,13 +33,13 @@ export default class RealtimeFaultProvider {
     }
 
     subscribe(domainObject, callback) {
-        const globalUnsubscribe = this.#openmct.telemetry.subscribe(
+        const globalUnsubscribe = this.realtimeProvider.subscribe(
```

Comment:
Shouldn't the openmct API be able to find the previously registered provider and subscribe appropriately?

Reply:
The Open MCT API calls the provider's subscribe method, which is the one above. Perhaps I'm not understanding your question?